src/Components/NavBar/NavBar.tsx

```diff
@@ -92,6 +95,15 @@ export const NavBar: React.FC = track(
     }
   }, [isMobile])
 
+  // Prefetch most clicked links.
+  // See: https://artsy.slack.com/archives/C05EEBNEF71/p1726074422149369?thread_ts=1726073316.088559&cid=C05EEBNEF71
+  useEffect(() => {
+    const PREFETCH_NAVBAR_LINKS = ["/artists", "/collect"]
+
+    PREFETCH_NAVBAR_LINKS.forEach(prefetch)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
🧹 ? 😬

Reply:
Heh, I am an anti exhaustive-deps person. I have seen more subtle bugs by following the guidance than without. (Deps are something one should be extremely deliberate with.)

Reply:
Usually it's a sign that we forgot something like memoize a function signature before using it in a useEffect, dealt with some bugs in the past because ppl forgot to put the deps there
I know @dzucconi shares some thoughts on it too

Reply:
If u wanna just do something once on mount u can use: https://github.com/artsy/force/blob/main/src/Utils/Hooks/useOnce.ts

Reply:
> forgot something like memoize a function signature before using it in a useEffect

I'm just into minimalism in this regard. I don't think wiring up basic things with `useCallback` and so on is necessary. It just adds complexity overhead. That's not to say those things don't have a purpose, but I'd rather skip the linter than satisfy it by adding another layer of (unnecessary) abstraction. 

BUT I like that useOnce 👍 Didn't know about it. Will use.

Reply:
I feel it, the code becomes verbose and the gain is none! I hope they come soon with react Forget that could change this way